l10n_it_withholding_tax/models/withholding_tax.py

```diff
@@ -126,6 +126,7 @@ def _check_date(self):
                 where.append("((date_start<='%s') or (date_start is null))" %
                              (self.date_stop,))
 
+            # pylint: disable=sql-injection
```

Comment:
Questo SQL non è molto leggibile, sarebbe meglio usare l'ORM invece di tacciare `lint`

Reply:
Sapresti come?

Reply:
No, appena ho tempo di guardarci provo a proporti una soluzione.
Nel frattempo secondo me è meglio che `lint` continui a sollevare il problema.

Reply:
Secondo me è meglio avere la build  verde e sbloccare ocabot, visto che il codice è in produzione da anni

Reply:
@eLBati ho fatto https://github.com/eLBati/l10n-italy/pull/101 per usare ORM

Reply:
@SimoRubi grazie, merged